### PR TITLE
feat: Add Terraform version check to cloud cli install script(s)

### PR DESCRIPTION
**Pull request by rmoles**
https://lacework.atlassian.net/browse/ALLY-1087

Signed-off-by: Ross <ross.moles@lacework.net>

### CODE REVIEW

File: aws/shell_startup.sh [hunk 1]
```diff
@@ -5,25 +5,50 @@ set -e
...
     fi
 fi
 
 # Install Terraform if it is not installed
-if [ ! -f "$HOME/bin/terraform" ]; then
-    curl https://releases.hashicorp.com/terraform/0.15.5/terraform_0.15.5_linux_amd64.zip --output $HOME/bin/terraform.zip
-    unzip $HOME/bin/terraform.zip -d $HOME/bin
-    rm $HOME/bin/terraform.zip
+if ! type terraform 2>&1; then
+    curl https://releases.hashicorp.com/terraform/0.15.5/terraform_0.15.5_linux_amd64.zip --output "$HOME"/bin/terraform.zip
```
[the remark this answers is not in the document]
Comment (rmoles) [reply aws/shell_startup.sh:24]: @dmurray-lacework shall I change to 1.2.5?

[the remark this answers is not in the document]
Comment (dmurray-lacework) [reply aws/shell_startup.sh]: I don't think we "officially" support 1.2.5 yet. Our Test pipeline is only up to 1.1 atm.

[the remark this answers is not in the document]
Comment (dmurray-lacework) [reply aws/shell_startup.sh]: Yeh, it's likely we wouldn't have any issues. But might be confusing that our docs list these versions, https://docs.lacework.com/terraform-for-lacework-overview#terraform-version-support
and then we install something else. If we can guarantee that >1.0.0 is always going to work we could amend the docs and always grab 'latest' TF version

[the remark this answers is not in the document]
Comment (dmurray-lacework) [reply aws/shell_startup.sh]: There is a ticket for this https://lacework.atlassian.net/browse/ALLY-1007 it's just a case of updating the docs, test pipeline, then updating the tf modules.

**Review by dmurray-lacework — APPROVED** [review]

**Review by afiune — APPROVED** [review]
tenor-242088441

**Pull request merged**